### PR TITLE
Makes Quad-Sec only work inside /area/security

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request
Makes quad-sec only have its healing effects apply in security areas
## Why It's Good For The Game
Security uses quad-sec to powergame mid-combat because it has no downsides that other cobbychems have to prevent powergaming.

## Changelog
:cl:
balance: Quadruple Security now only heals inside security areas
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -1558,9 +1558,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	//Securidrink in line with the Screwdriver for engineers or Nothing for mimes
 	var/obj/item/organ/liver/liver = M.getorganslot(ORGAN_SLOT_LIVER)
 	if(liver && HAS_TRAIT(liver, TRAIT_LAW_ENFORCEMENT_METABOLISM))
-		M.heal_bodypart_damage(brute = 1, burn = 1)
-		M.adjustBruteLoss(-2,0)
-		. = 1
+		if(istype(get_area(M), /area/security)) // Skyrat edit , it checks for area now - Start
```
Comment (KathrinBailey) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:1561]:
```suggestion
		if(istype(get_area(M), /area/security)) // SKYRAT EDIT : Checks for area.
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -1558,9 +1558,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+			M.heal_bodypart_damage(brute = 1, burn = 1)
+			M.adjustBruteLoss(-2,0)
+			. = 1
+		return ..() // Skyrat edit - end
```
Comment (KathrinBailey) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
		return ..() // SKYRAT EDIT: End - This line and the above few have only indentation changes.
```

**Review by KathrinBailey — CHANGES_REQUESTED** [review]
The comments are a tad confusing on this, you haven't changed the code below in any way other than indentation, which you should clarify. If you do this it's an EZMERGE. It's just that the way you did it implies there may be value changes to the healing effects.

**Comment by MLGTASTICa** [comment]
> The comments are a tad confusing on this, you haven't changed the code below in any way other than indentation, which you should clarify. If you do this it's an EZMERGE. It's just that the way you did it implies there may be value changes to the healing effects.

Commited your suggestion to the comments, should be enough right?

**Comment by Draganfrukts** [comment]
I think it should have an effect in the bar too, I mean, kinda nullifies anything social about asking it to get it done for you

**Comment by MLGTASTICa** [comment]
> I think it should have an effect in the bar too, I mean, kinda nullifies anything social about asking it to get it done for you

Bar is usually too central to the station and there is no reason to have it heal in the bar , if you are there to RP you don't need the healing , otherwise seccies will just sprint or fight in the bar because they know they can heal.

**Comment by Gandalf2k15** [comment]
I'll accept.

**Pull request merged**